app/workers/chase_references.rb

```diff
@@ -1,75 +1,87 @@
-# An email is sent to the candidate at 7 days, 14 days and 28 days
-# An email is sent to the referee at 7 days, 21 days and 28 days
+# An email is sent to candidates at 9 days, 16 days and 30 days
+# An email is sent to referees at 7 days, 14 days and 28 days
```

Comment:
tbh i'd just delete this comment. the code is perfectly clear, and the comment is just one more thing to maintain.

Reply:
Done!